multi-int-store/src/main/java/com/connexta/multiintstore/config/ApplicationConfiguration.java

```diff
@@ -42,4 +43,16 @@ public EndpointUrlRetrieve endpointUrlRetrieve() {
   public RestTemplate restTemplate(RestTemplateBuilder builder) {
     return builder.errorHandler(new DefaultResponseErrorHandler()).build();
   }
+
+  @Bean
+  public CommonsRequestLoggingFilter requestLoggingFilter() {
```

Comment:
✏️ The request logs come out a bit messy. Is there a way to "pretty print" them?

Reply:
Not easily. The log format is one message = one line. Logs are a bad solution for hierarchical data.
@oconnormi has a plan to stand up an ELK stack to parse and display logs. Once that happens, I believe we will be able to extract and pretty print the HTTP requests.